wiki/UI-component/Lightbox-media-viewer.md

```diff
@@ -85,6 +84,16 @@ Renders an [[Image]] or [[Video]] depending on what the adopters passes.
 | `2.1.1` | [[Image]]  | If an [[Image]] is provided, render it as the main media piece.  |
 | `2.1.2` | [[Video]] `ID` | If a [[Video]] `ID` is provided use the [[Video service]] to get [[Video]] data and render |
 
+###### Events
+
+The following events are only applicable to mobile devices.
+
+| event  | description  |
+|:-----------|:-------------|
+| on tap | Triggers isolated media view on mobile. This view allows user to pinch, zoom and drag. |
```

Comment:
It's more events than just tap here. Each of the events you called out in this line needs its own row. Like ping, drag to name a couple. We have to be very explicit here, and make sure we get all of them.